src/index.js

```diff
@@ -1,217 +1,41 @@
-import fsApi from 'fs';
-import path from 'path';
-import events from 'events';
+import scheduler from 'node-schedule';
 
-import config from 'config';
+import CGUs from './app/index.js';
+import * as logger from './logger/index.js';
+import Notifier from './notifier/index.js';
 
-import consoleStamp from 'console-stamp';
+(async () => {
+  try {
```

Comment:
Could we please prefer to wrap with `try` only the parts where we actually expect an exception? This way we don't `catch` stuff that we don't expect 🙂